src/ceph-volume/ceph_volume/tests/functional/lvm/playbooks/test_filestore.yml

```diff
@@ -8,14 +8,34 @@
         name: ceph-osd@2
         state: stopped
 
+    - name: stop ceph-osd@0 daemon
+      service:
+        name: ceph-osd@2
```

Comment:
This needs to be ``ceph-osd@0``

Reply:
woah, great catch, not sure how I missed that one